apps/design-land/src/app/checkbox/checkbox.module.ts

```diff
@@ -1,61 +1,16 @@
-import {
-  NgModule,
-  ComponentFactoryResolver,
-  Injector,
-} from '@angular/core';
-import {
-  DaffCheckboxModule,
-  DaffButtonModule,
-} from '@daffodil/design';
-import { ReactiveFormsModule } from '@angular/forms';
 import { CommonModule } from '@angular/common';
+import { NgModule } from '@angular/core';
 
-import { CheckboxComponent } from './checkbox.component';
-import { DesignLandCheckboxRoutingModule } from './checkbox-routing.module';
-import { CHECKBOX_EXAMPLES } from '@daffodil/design/checkbox/examples';
-import {
-  createCustomElement,
-  NgElementConstructor,
-} from '@angular/elements';
 import { DesignLandExampleViewerModule } from '../core/code-preview/container/example-viewer.module';
-
-
-
-export interface CustomClassElement<T> {
-    element: NgElementConstructor<T>;
-    class: T;
-}
+import { DesignLandCheckboxRoutingModule } from './checkbox-routing.module';
 
 @NgModule({
-
-  declarations: [
-    CheckboxComponent,
-    ...CHECKBOX_EXAMPLES,
-  ],
   imports: [
     DesignLandExampleViewerModule,
     DesignLandCheckboxRoutingModule,
-    DaffCheckboxModule,
-    ReactiveFormsModule,
     CommonModule,
   ],
 })
 export class CheckboxModule {
```

Comment:
Can we update this to `DesignLandCheckboxModule` and do it in the component too since you started doing that to other components in the PR?